flank-scripts/src/test/kotlin/flank/scripts/ci/nexttag/NextReleaseTagGeneratorTest.kt

```diff
@@ -23,4 +24,88 @@ class NextReleaseTagGeneratorTest {
             assertThat(generateNextReleaseTag("v20.08.1")).isEqualTo("v20.09.0")
         }
     }
+
+    companion object {
+        @JvmStatic
+        @BeforeClass
+        fun setUp() {
+            runCatching {
+                mockkStatic(LocalDate::class) {
+                    // Mockk probably has a bug because sometimes first call of
+                    // every { LocalDate.now() } is failing with strange error:
+                    /*
+                  every/verify {} block were run several times. Recorded calls count differ between runs
+Round 1: class java.time.LocalDate.of(-999999999, 1, 1), class java.time.LocalDate.of(999999999, 12, 31), class java.time.LocalDate.now()
+Round 2: class java.time.LocalDate.now()
+io.mockk.MockKException: every/verify {} block were run several times. Recorded calls count differ between runs
+Round 1: class java.time.LocalDate.of(-999999999, 1, 1), class java.time.LocalDate.of(999999999, 12, 31), class java.time.LocalDate.now()
+Round 2: class java.time.LocalDate.now()
```

Comment:
do we explicitly need to have the entire stacktrace here taking up 40+ lines? Possibly just a `...`for the less important aspects would suffice.

Reply:
FIXED